src/backend/routes/tankRoutes.js

```diff
@@ -118,7 +118,7 @@ router.get('/getBotTanks', tankController.getBotTanks);
 // Takes an array of tank ids
 // returns the tank objects
 router.get('/getTanksById', [
-	check('array', 'An array of MongoIds is required.').isArray().isMongoId()
+	check('array', 'A JSON array of MongoIds is required.').isJSON()
 ], tankController.getTankById);
```

Comment:
I didn't notice this the first time, but getTanksById maps to getTankById? Sounds like a confusing name unless there is something going on here that I'm not understanding...

Reply:
It's just a reference to the function in tankController. If we want them to match it's just a matter of matching the names

Reply:
In the grand scheme of things it doesn't affect functionality